beacon-chain/light-client-network.md

```diff
@@ -1,7 +1,7 @@
-# Beacon Chain Light Client Network
+# Beacon Chain Network
```

Comment:
If we rename the network, the name of the file should also change.

Reply:
Perhaps just leave the name as Beacon Chain LC network for now.

Reply:
I think I'm going to keep for now the network name  `Beacon Chain Network`  because the data that we are going to store is not only light client data. I'll rename the file in a separate PR to keep the current changes clean.